README.md

```diff
@@ -10,26 +10,27 @@ Sie bauen Websites und Apps – konkret deren Frontend. Das Frontend ist das, wa
 Frontender stehen im Prozess zwischen Designer und Backender und haben – ober besser sollten – für eine reibungsarme Zusammenarbeit überschneidende Kompetenzen mit beiden Berufen haben.
 
 ## Grobe «Skizze» des Fähigkeitenkatalogs
-- kann HTML, CSS, JS
-- *kann API (zu präzisieren).*
-- kann SVG.
-- *kann das «Drumherum» (Sass, Pug, ES6/Babel und wie sie alle heissen – zu präzisieren)*
+- kann HTML und versteht die den Unterschied zwischen Semantik und visueller Darstellung.
+- kann CSS und kennt gebräuchliche Präpozessoren.
+- kann JS und kennt gebräuchliche Präpozessoren.
+- kann APIs konsumieren (RESTful und GraphQL) und kennt deren Eigenheiten.
+- kann Vektorbearbeitung und versteht deren Dateiformate (SVG).
+- kann Bildbearbeitung und versteht deren Dateiformate (SVG).
 - *kann «Legacy-Technologien» (zu präzisieren, Beispiel PHP).*
 - kann Git.
-- findet sich im Dschungel der Technologien zurecht, beurteilt Trends, reflektiert, hält sich Up-to-date.
-- kennt die populären CMS und kann deren Templates bauen.
-- kann sich schnell innert nützlicher Frist in neue, verwandte Technologien einarbeiten.
+- findet sich im Dschungel der Technologien zurecht, beurteilt Trends, reflektiert
+- kann Wissen selbstständig erschliessen und für sich nutzbar machen.
+- kennt die wichtige CMS und kann dafür Templates bauen.
+- kann verwandte Technologien und Arbeitsweisen erkennen und adaptieren.
 - *versteht Prozesse des Backends (zu präzisieren).*
-- *ist fit in Typografie (zu präzisieren).*
+- kennt die typografischen Eigenheiten von interaktiven Inhalten und kann diese, dem Ausgabemedium angepasst, gestalten.
 - *ist fit in Design (zu präzisieren).*
```

Comment:
Kann Designkonzepte interpretieren und weiterentwickeln.

Reply:
👌 habe ich nun in meinen PR übernommen. kannst du aber natürlich auch gerne selber als commit anfügen, damit die Quelle beim stimmt. fb3c6484603fa0026914fbdc04430c1cc3e659b7